core/frontend/src/FrontendHubAccess.ts

```diff
@@ -6,71 +6,22 @@
  * @module HubAccess
  */
 
-import { GuidString, IModelStatus } from "@bentley/bentleyjs-core";
-import { addCsrfHeader, ChangeSet, ChangeSetQuery, IModelClient, IModelHubClient, VersionQuery } from "@bentley/imodelhub-client";
-import { IModelError, IModelVersion } from "@bentley/imodeljs-common";
+import { GuidString } from "@bentley/bentleyjs-core";
+import { IModelVersion } from "@bentley/imodeljs-common";
 import { AuthorizedClientRequestContext } from "@bentley/itwin-client";
-import { IModelApp } from "./IModelApp";
 
-/** @internal */
+/** @public */

```

Comment:
no, please remove this and use `ChangesetId`  from imodeljs-common

Reply:
updated.